hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMStateMachine.java

```diff
@@ -175,6 +176,14 @@ private Message process(final SCMRatisRequest request) throws Exception {
     }
   }
 
+  @Override
+  public void notifyLogFailed(Throwable ex,
+                              RaftProtos.LogEntryProto failedEntry) {
+    // Should we add retry here?
+    LOG.error("SCM statemachine appendLog failed, entry: {}", failedEntry);
```

Comment:
it is better off showing why this IO error happens in the log message?
for example , the error number and the corresponding message